libcontainer/cgroups/fs/memory.go

```diff
@@ -269,3 +284,76 @@ func getMemoryData(path, name string) (cgroups.MemoryData, error) {
 
 	return memoryData, nil
 }
+
+func getPageUsageByNUMA(cgroupPath string) (cgroups.PageUsageByNUMA, error) {
+	stats := cgroups.PageUsageByNUMA{}
+
+	file, err := os.Open(path.Join(cgroupPath, "memory.numa_stat"))
+	if err != nil {
+		return stats, err
+	}
+
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		var statsType string
+		statsByType := cgroups.PageStats{Nodes: map[uint8]uint64{}}
+		columns := strings.SplitN(scanner.Text(), numaStatColumnSeparator, numaStatMaxColumns)
+
+		for _, column := range columns {
+			pagesByNode := strings.SplitN(column, numaStatKeyValueSeparator, 2)
+			if len(pagesByNode) != 2 {
+				return cgroups.PageUsageByNUMA{}, fmt.Errorf("wrong data format, found %d fields instead of 2", len(pagesByNode))
+			}
+
+			if isNUMANode.MatchString(pagesByNode[0]) {
```

Comment:
Consider introducing new consts e.g numaStatKey = 0  and numaStatValue =1 to use  `pagesByNode[numaStatKey]` and `pagesByNode[numaStatValue]`  instead of ` pagesByNode[0]`  and `pagesByNode[1]`